### ISSUE TITLE
Video streaming format

### ISSUE THREAD

**Issue by zakiloki0** [telegramdesktop/tdesktop#17262]
I have made some home videos to share with a group of my family, so that they can see them in streaming, but it seems that Telegram does not play it. What is the ideal format for videos?

**Comment by Aokromes** [comment]
> I have made some home videos to share with a group of my family, so that they can see them in streaming, but it seems that Telegram does not play it. What is the ideal format for videos?

mp4.

**Comment by zakiloki0** [comment]
> > I have made some home videos to share with a group of my family, so that they can see them in streaming, but it seems that Telegram does not play it. What is the ideal format for videos?
> 
> mp4.

Yes I know is MP4 container, but i mean what codecs are better for Telegram.

**Comment by ilya-fedin** [comment]
Here you can see enabled codecs in ffmpeg build for tdesktop on Windows: https://github.com/desktop-app/patches/blob/master/build_ffmpeg_win.sh#L14-L112

On phones, it depends on device vendor and which codecs it bought

**Comment by zakiloki0** [comment]
> Here you can see enabled codecs in ffmpeg build for tdesktop on Windows: https://github.com/desktop-app/patches/blob/master/build_ffmpeg_win.sh#L14-L112
> 
> On phones, it depends on device vendor and which codecs it bought

Thanks!

**Comment by zakiloki0** [comment]
I close this.
